back-end/src/main/resources/application-dev.properties

```
@@ -1,4 +1,5 @@
 spring.config.import=./application-db.properties
 spring.jpa.database-platform=org.hibernate.dialect.MariaDBDialect
 spring.jpa.hibernate.ddl-auto=create
```

```suggestion
spring.jpa.hibernate.ddl-auto=validate 혹은 none
```
dev, prod 설정을 바꿀 수 있을 것 같아요!

---

저희 마리아 DB로 전환하면서 데이터가 다 들어가 있나요?
들어가 있으면 validate로 전환하겠습니다

---

기존 create 하고 더미랑 이후 작업하면서 추가했다면 추가되어있어요:)

---

`validate`로 변경하였습니다.